recipes/pystem/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "pystem" %}
+{% set version = "0.0.21" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 6f1365702a0fde18837143f1a5777795c4cd89b3cbf8b05a5a8cdc26a04180e8
+
+build:
+  number: 0
+  skip: true  # [win]
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - pip
+    - python
+    - fftw
+    - numpy >=1.17.0
```

Comment:
This is [using the numpy C API](https://github.com/NingWang1990/pySTEM/blob/0f2de23d790d9723dd6175663282279843c0a73a/pystem/stemdescriptor2.cpp#L7) so you need to follow:

https://conda-forge.org/docs/maintainer/knowledge_base.html#linking-numpy

I'd also encourage you to drop the version pin and use the global numpy pinning for the reasons stated in https://github.com/conda-forge/staged-recipes/pull/12043#discussion_r448363865